common/Resources/ti.internal/aca.js

```diff
@@ -8,8 +8,6 @@
  * This allows ACA to be the first module to load on startup.
  */
 
-try {
-	import('com.appcelerator.aca');
-} catch (e) {
-	// Could not load module, silently ignore exception.
-}
+import('com.appcelerator.aca').catch(e => {
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/aca.js;
  Line: 11;
-->

- :warning: common/Resources/ti.internal/aca.js line 11 – 'e' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)